src/PointwiseFunctions/Hydro/Tags.hpp

```diff
@@ -174,6 +174,12 @@ struct SpecificInternalEnergy : db::SimpleTag {
   using type = Scalar<DataType>;
 };
 
+/// The lower index spatial for velocity \f$u_i\f$.
```

Comment:
try again :). I'd suggest `The spatial components of the four-velocity one-form`